src/linear_solver/NNLS_solver.h

```diff
@@ -167,6 +184,21 @@ class NNLS_solver
 
     /// Moves the column at idx into the inactive set (updating the index_set, Z, P, and numInactive_)
     void moveToInactiveSet(int idx);
+
+    /// @brief Re-allocates solution to multiple cores
+    /// @param c Epetra_vector to reallocate on multiple cores
+    /// @return An Epetra_vector with the data of c distributed on multiple cores
+    Epetra_Vector allocateToMultipleCores(const Epetra_Vector &c);
+
+    /// @brief Allocates the incoming A Matrix to a single core
+    /// @param A Epetra_CrsMatrix to reallocate on a single core
+    /// @return An Epetra_CrsMatrix with the data of A distributed on a single core
+    Epetra_CrsMatrix allocateMatrixSingleCore(const Epetra_CrsMatrix &A);
```

Comment:
Renaming to `allocateMatrixToSingleCore` and `allocateVectortoSingleCore` would be more descriptive.

Reply:
I have renamed all three of the epetra allocation functions to have the more descriptive names in commit eb47054